articles/protocols/saml/saml-configuration/troubleshoot/auth0-as-idp.md

```diff
@@ -0,0 +1,100 @@
+
+The following sections describe how to check each stage and how to identify if there are any issues with a given stage.
+
+## Issue: User Successfully Logs In With the IdP, but a Successful Login Event Does *Not* Show Up in Auth0 Logs
+
+* If you're using an Auth0 Database Connection:
+  * Check that the user exists and the entered password is correct.
+  * Disable your [rules](/rules) temporarily to make sure that nothing is interfering with the login process.
+  * If you've enabled [multifactor authentication](/multifactor-authentication), disable it temporarily to make sure that it is not interfering with the login process.
+
+* If you're using an Auth0 Database Connection **or** a remote SAML connection:
+  * Check that the SAML Connection works by [using **Try** to run a Connection test](#issue-the-idp-login-page-doesn-t-display).
+
+## Issue: User Successfully Logs In With the IdP, but a Successful Login Event Does *Not* Show Up in Auth0 Logs
+
+* If you're using an Auth0 Database Connection:
+  * Check that the user exists and the entered password is correct.
+  * Disable your [rules](/rules) temporarily to make sure that nothing is interfering with the login process.
+  * If you've enabled [multifactor authentication](/multifactor-authentication), disable it temporarily to make sure that it is not interfering with the login process.
+
+* If you're using an Auth0 Database Connection **or** a remote SAML connection:
+  * Check that the SAML Connection works by [using **Try** to run a Connection test](#issue-the-idp-login-page-doesn-t-display).
+
+## Issue: User Successfully Logs In With the IdP, a Successful Login Event Shows Up in Auth0 Logs, but the User's Profile Attributes are Incorrect
+
+If the user:
+
+* Appears to log in successfully
+* The [Logs](${manage_url}/#/logs) and [Users](${manage_url}/#/users) pages in the Auth0 Dashboard should successful login events
+
+The next step is to check that the user's profile contains the necessary user profile attributes.
+
+### Checking the User Profile
+
+1. After logging in to the [Auth0 Dashboard, navigate to *Users*](${manage_url}/#/users).
+2. Find and click on the specific user to open up their profile. If there are multiple rows for a given user, be sure to open up the record associated with the SAML Connection.
+3. On the user's profile, you can view their details in one of two ways. You can use the *Details* tab or the *Raw JSON* tab. This shows you what attributes Auth0 has received from the identity provider.
+
+If an attribute is missing, check with the identity provider to confirm that it has the attribute and that it is returning that attribute to Auth0.
+
+## Issue: User Successfully Logs In With the IdP, a Successful Login Event Shows Up in Auth0 Logs, and the User's Profile Attributes are Correct, but the User Cannot Access the Application
+
+* Check to see if the user's Auth0 profile populated correctly:
```

Comment:
Same comment as earlier, add an empty line between bullets, the text looks too stuffed.